mix_helpers.exs

```diff
@@ -252,11 +252,28 @@ defmodule Mix.Appsignal.Helper do
   end
 
   defp download_options do
+    default_cacert_file_path = priv_path("cacert.pem")
+
+    cacert_file =
+      case check_cacert_access(default_cacert_file_path) do
+        :ok ->
+          default_cacert_file_path
+
+        {:error, message} ->
+          Logger.warn(
+            "The cacert file path: #{default_cacert_file_path} is not accessible. " <>
+              "Reason: #{inspect(message)}. " <>
+              "Using system defaults instead."
+          )
```

Comment:
This should be logged in the download report, but we don't have a field for "warnings" or "messages". Let's keep it like this for now. Can you create an issue for moving this to the download report file?

Looking at the code it's going to a hassle to get this non-error message all the way up to the top again. This is buried quite deep. It's probably easier to do the check twice, earlier in another location so that it's easier to log to the download report file.